buildspec_sync.yml

```diff
@@ -60,8 +60,8 @@ phases:
       - export AWS_SECRET_ACCESS_KEY=
       - export AWS_SESSION_TOKEN=
 
-      # Verify the publishing on Public ECR and Dockerhub
-      - './scripts/publish.sh cicd-verify stable public-repo'
```

Comment:
the existing `public-repo` argument isn't used anywhere?

Reply:
Previously, we need it as we need 3 arguments here to check if we are in which case. But now, I've deleted it and add the AWS_REGION as another arg